extensions/blocks/testimonial/editor.scss

```diff
@@ -0,0 +1,121 @@
+.wp-block-jetpack-testimonial {
+    display: flex;
+    flex-direction: column;
+}
+
+.wp-block-jetpack-testimonial__content {
+    .is-aligned-left & {
+        text-align: left;
+    }
+    .is-aligned-center & {
+        text-align: center;
+    }
+    .is-aligned-right & {
+        text-align: right;
+    }
+}
+
+.wp-block-jetpack-testimonial__author {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    
+    .is-aligned-right & {
+        justify-content: flex-start;
+    }
+    
+    .is-aligned-center & {
+        justify-content: center;
+    }
+    
+    .is-aligned-right & {
+        justify-content: flex-start;
+        flex-direction: row-reverse;
+        text-align: right;
+    }
+}
+
+.wp-block-jetpack-testimonial__meta {
+    display: flex;
+    flex-direction: column;
+    color: #999;
+    font-size: 80%;
```

Comment:
Should you use `em`s here?

Reply:
Another placeholder, I don't even know the exact styles we will use yet so I just put random percentages.